db/migrate/20190619132408_change_closed_default_in_school.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeClosedDefaultInSchool < ActiveRecord::Migration[5.2]
+  def change
+    change_column_default :schools, :closed, false
```

Comment:
until it's merge into development / master i recommend you to update `db/migrate/20190619114036_create_schools.rb` migration :)